analyzers/src/SonarAnalyzer.Common/Facade/SyntaxFacade.cs

```diff
@@ -34,6 +34,7 @@ public abstract class SyntaxFacade<TSyntaxKind>
         public abstract SyntaxToken? InvocationIdentifier(SyntaxNode invocation);
         public abstract SyntaxNode NodeExpression(SyntaxNode node);
         public abstract SyntaxToken? NodeIdentifier(SyntaxNode node);
+        public abstract string GetStringTextValue(SyntaxNode node);
```

Comment:
As it expects any Node as input (contrary to InvocatonIdentifier that expect Invocation as an input)
```suggestion
        public abstract string NodeStringTextValue(SyntaxNode node);
```